docs/source/actions.rst

```diff
@@ -79,20 +79,22 @@ Available runners
 The environment in which the action runs is specified by the runner.
 Currently the system provides the following runners:
 
-1. ``local-script`` - This is the local runner. Actions are implemented as
+1. ``local-shell-cmd`` - This is the local runner. This runner executes
+   a Linux command on the same host where |st2| components are running.
+2. ``local-shell-script`` - This is the local runner. Actions are implemented as
    scripts. They are executed on the same hosts where |st2| components are
    running.
-2. ``remote-script`` - This is a remote runner. Actions are implemented as scripts.
+3. ``remote-script`` - This is a remote runner. Actions are implemented as scripts.
```

Comment:
This should also be remote-shell-script and looks like there is also a missing entry for remote-shell-cmd.